.github/workflows/ee-ep-merge-end-to-end.yml

```diff
@@ -0,0 +1,72 @@
+name: 'CI: EE - EP Merge - Integration Test'
+
+on:
+  # Allow manual triggering
+  workflow_dispatch:
+
+  # Allow being called by another GitHub Action
+  workflow_call:
+
+env:
+  COMPOSE_PROFILES: ''
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout source code'
+        uses: actions/checkout@v3
+
+      - name: 'Build the images'
+        uses: ./.github/actions/build-images
+
+      - name: 'Start the containers'
+        run: |
+          source scripts/setenv.sh
+
+          # create basic auth token for RabbitMQ and export to github environment
+          BASIC_AUTH=$(echo "${RABBITMQ_PLACEHOLDERS_USERNAME}:${RABBITMQ_PLACEHOLDERS_USERPASSWORD}" | base64)
```

Comment:
For some reason when I run this locally, I get "401 Unauthorized" unless I add a `-n` flag to `echo`. This removes a trailing newline character that was getting base64-encoded, changing the authorization header from `Basic Z3Vlc3Q6Z3Vlc3QK` to `Basic Z3Vlc3Q6Z3Vlc3Q=` (note `=` is the last char).

I can confirm when using curl's `-u` flag for sending basic auth credentials that it uses the string without the trailing newline: 
```
$ curl -v -u "guest:guest" http://localhost:15672/api/vhosts
*   Trying ::1:15672...
* Connected to localhost (::1) port 15672 (#0)
* Server auth using Basic with user 'guest'
> GET /api/vhosts HTTP/1.1
> Host: localhost:15672
> Authorization: Basic Z3Vlc3Q6Z3Vlc3Q=
> User-Agent: curl/7.74.0
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
``` 

Not sure why CI isn't affected by this, but wanted to call it out just in case.

Reply:
That is strange that it still works in the integration test!

Reply:
Looks like the curl [action](https://github.com/indiesdev/curl/blob/13079d5a2799e82d372ed54ad51384820e96a832/src/requestconf.ts#L43) im using calls trim() on the base64 encoded string.